pkg/eks/tasks.go

```diff
@@ -15,6 +18,37 @@ type clusterConfigTask struct {
 	call func(*api.ClusterConfig) error
 }
 
+
+func endpointsEqual(a, b api.ClusterEndpoints) bool {
+	ajson, err := json.MarshalIndent(a, "", "\t")
+	if err != nil {
+		return false
+	}
+	bjson, err := json.MarshalIndent(b, "", "\t")
+	if err != nil {
+		return false
+	}
+	return string(ajson) == string(bjson)
+}
+
+//DefaultEndpointsMsg returns a message that the EndpointAccess is the same as the default
+func DefaultEndpointsMsg(cfg *api.ClusterConfig) string {
+	return fmt.Sprintf(
+		"Cluster API server endpoint access will use default of {Pulic: true, Private false} for cluster %q in %q", cfg.Metadata.Name, cfg.Metadata.Region)
+}
+
+//CustomEndpointsMsg returns a message indicating the EndpointAccess given by the user
+func CustomEndpointsMsg(cfg *api.ClusterConfig) string {
+	return fmt.Sprintf(
+		"Cluster API server endpoint access will use provided values {Public: %v, Private %v} for cluster %q in %q", *cfg.VPC.ClusterEndpoints.PublicAccess, *cfg.VPC.ClusterEndpoints.PrivateAccess, cfg.Metadata.Name, cfg.Metadata.Region)
+}
+
+//UpdateEndpointsMsg gives message indicating that they need to use eksctl utils to make this config
+func UpdateEndpointsMsg(cfg *api.ClusterConfig) string {
+	return fmt.Sprintf(
+		"you can update endpoint access with `eksctl utils update-cluster-api-access --region=%s --name=%s --private-access bool --public-access bool", cfg.Metadata.Region, cfg.Metadata.Name)
```

Comment:
nit: consistency with `--private-access=bool --public-access=bool`. also there seems to be a missing closing backtick

Reply:
fixed.